packages/react-components/src/templates/WorkingGroupAbout.tsx

```diff
@@ -1,18 +1,16 @@
-import { TeamResponse } from '@asap-hub/model';
+import { WorkingGroupResponse } from '@asap-hub/model';
 import { css } from '@emotion/react';
 
 import { Card, Headline3, Link, Subtitle } from '../atoms';
 import { createMailTo } from '../mail';
 import { Collapsible } from '../molecules';
+import { DeliverablesCard } from '../organisms';
 import { perRem } from '../pixels';
 
 type WorkingGroupAboutProps = Pick<
-  TeamResponse,
-  'members' | 'pointOfContact'
-> & {
-  description: string;
```

Comment:
Not sure why we are injecting a different description per group. This seems like a mistake but 🤷